### PR TITLE
nit: update statement about OpenSSL and Node.js 10.x

### PR DESCRIPTION
**Pull request by brodybits**

### CODE REVIEW

File: OpenSSL-Strategy.md [hunk 1]
```diff
@@ -312,8 +312,8 @@ late 2017.
 Even though OpenSSL 1.1.0 is only supported until 2019-09-11, the API shift was
 an important stage in Node.js' adaptation.
 
-Openssl 1.1.0 is currently included in Node.js 10.x, but Node.js is expected to
-be updated to include OpenSSL 1.1.1 in the upcoming 10.16.0 release.
+Openssl 1.1.0 was included in Node.js 10.x in the past; Node.js was
```
Comment (brodybits) [inline OpenSSL-Strategy.md:315]: put a word back ??

```suggestion
Openssl 1.1.0 was included in Node.js 10.x in the past; but Node.js was
```

**Review by mhdawson — APPROVED** [review]
LGTM

File: OpenSSL-Strategy.md (hunk 1 above)
Comment (Trott) [reply OpenSSL-Strategy.md:315]: I think it's fine without the "but" although I guess it's OK with "but" too? Semicolons are sometimes more clear as periods/full stops.

```suggestion
Openssl 1.1.0 was included in Node.js 10.x in the past. Node.js was
```

**Review by Trott — APPROVED** [review]
LGTM with or without semicolon and with or without "but".

**Comment by danbev** [comment]
Landed in 5a5963a8b6868c6f8ab7818b833e65a266063132.

**Pull request closed without merging**